File: SilksyClient/src/app/model/orderCreateParams.ts
import { Address } from "./address";

export class OrderCreateParams {
     
    constructor(paymentIntentId: string, address: Address) {
        this.paymentIntentId = paymentIntentId;
        this.shippingAddress = address
    }
    
    paymentIntentId: string;
    shippingAddress: Address;
}

File: SilksyAPI/Helpers/OrderCreateParams.cs
using SilksyAPI.Dto;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SilksyAPI.Helpers
{
    public class OrderCreateParams
    {
        public string PaymentIntentId { get; set; }

        public AddressDto ShippingAddress { get; set; }
    }
}
